doc/types/enums_and_composites.md

```diff
@@ -52,11 +52,11 @@ using (var reader = cmd.ExecuteReader()) {
 ```
 
 
-Note that your PostgreSQL enum and composites types (`pg_enum_type` and `pg_composite_type` in the sample above) must be defined in your database before the first connection is created (see `CREATE TYPE`). If you're creating PostgreSQL types within your program, call `NpgsqlConnection.ReloadTypes()` to make sure Npgsql becomes properly aware of them.
+Note that your PostgreSQL enum and composites types (`enum_type` and `composite_type` in the sample above) must be defined in your database before the first connection is created (see `CREATE TYPE`). If you're creating PostgreSQL types within your program, call `NpgsqlConnection.ReloadTypes()` to make sure Npgsql becomes properly aware of them.
```

Comment:
The sample above has `some_enum` and `some_composite` as the types, not `enum_type` and `composite_type`... This still needs to be cleaned up...

Reply:
I personally think suffixing `_type` makes it clearer, otherwise people may get confused and think we're referring to a column.

Reply:
I've added the `_column` suffix like it was done for the table. So there should be no ambiguity.